src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -50,56 +52,21 @@ const SetAclPermissionForm = () => {
     const permissions = event.target.setAclPerms.value
       ? { read: event.target.setAclPerms.value === 'Give' }
       : undefined;
-    const podUsername = event.target.setAclTo.value;
-
-    if (!podUsername) {
-      runNotification('Set permissions failed. Reason: Username not provided.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (getPodUrl(podUsername) === podUrl) {
-      runNotification(
-        'Set permissions failed. Reason: Current user Pod cannot change container permissions to itself.',
-        5,
-        state,
-        dispatch
-      );
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (permissions === undefined) {
-      runNotification('Set permissions failed. Reason: Permissions not set.', 5, state, dispatch);
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
-      return;
-    }
-
-    if (!docType) {
-      runNotification('Search failed. Reason: No document type selected.', 5, state, dispatch);
-      setTimeout(() => {
-        dispatch({ type: 'CLEAR_PROCESSING' });
-      }, 3000);
-      return;
-    }
+    const webIdToSetPermsTo = `${permissionState.podUrlToSetPermissionsTo}profile/card#me`;
 
     try {
-      await setDocAclPermission(session, docType, permissions, podUsername);
+      await setDocAclPermission(session, docType, permissions, podUrl, webIdToSetPermsTo);
 
       runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${username} for ${docType}.`,
+        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
+          permissionState.podUrlToSetPermissionsTo
+        } for ${docType}.`,
         5,
         state,
         dispatch
       );
     } catch (error) {
-      runNotification('Set permissions failed. Reason: File not found.', 5, state, dispatch);
+      runNotification('FAILED TO SET PERMISSIONS. REASON: File not found.', 5, state, dispatch);
```

Comment:
A bit curious about the reason behind the change in case?

Reply:
Just for matter of importance, like using different <h> tags...  I can switch it back if preferred. I'm not married to the idea.

Reply:
No preference for either, but was wondering if we could make the whole notification line uppercase if we're going to use this. Otherwise, we can revert this back.